### DISCUSSION TITLE
setting window title on macos

### DISCUSSION THREAD

**Discussion by cakir-enes** [neovide/neovide#2302] [Q&A] [answered]
there is this option supposed to work according to #1553 
```lua
if vim.g.neovide then
  vim.g.neovide_title = "neovidero"
send
```
in the end I just want to append launched folder name to it

**Comment by fredizzimo** [comment]
You can use this https://neovim.io/doc/user/options.html#'titlestring' to set the title

The suggestion was never implemented, because it was already possible do from within Neovim.

**Reply by cakir-enes** [reply]
Thanks but it did not work on macos :/

**Reply by falcucci** [reply, marked as answer]
it works.

try to enable it first and then set your custom title.

```
set title
set titlestring='test'
```
